components/BaseContentController.php

```diff
@@ -219,39 +217,15 @@ public function actionDeleteByContainer($containerId)
 
     public function actionAttachFiles($id)
```

Comment:
Should we remove this? And add to docs, to use ContentController instead?

Reply:
@luke- Please note this base action is used in 2 modules:

- https://github.com/humhub/tasks/blob/master/Events.php#L320
- https://github.com/humhub/calendar/blob/master/Events.php#L403

If the endpoints are already used by someone then it may be broken if we will remove this.
We can deprecate the method `BaseContentController::actionAttachFiles($id)` for now.

Reply:
@yurabakhtin Okay, I didn't see that.

I would then say that each content type (post, poll, calendar) must implement the file upload via BaseContentController.

- In that case it makes no sense if we keep `content/$id/attach-files'.

Can you change this again?